pkg/server/ca/manager/manager.go

```diff
@@ -31,8 +31,9 @@ import (
 )
 
 const (
-	publishJWKTimeout = 5 * time.Second
-	safetyThreshold   = 24 * time.Hour
+	publishJWKTimeout         = 5 * time.Second
+	safetyThresholdBundle     = 24 * time.Hour
+	safetyThresholdCAJournals = time.Hour * 24 * 14 // Two weeks
```

Comment:
so, we are going to remove journals, that has bundles with more than 2 weeks, 
I hear about some cases where SVIDs has 7 days ttl, so I suspect that those bundles has at least 1month,
and what happens if that is only a leaf from a nested spire? in that cases we'll have bundles with more than 1 month, 
Is it not possible that this can remove journals that are still active?

Reply:
I don't think so. How long is the TTL doesn't really matter for the pruning logic. This is about pruning CA journals that have all of the CAs expired since at least 2 weeks. This means that those CAs have been unusable for at least 2 weeks. They may have had a TTL of 6 months, but have expired two weeks ago, so they can be pruned.